### PR TITLE
Jcg/ada 11 signin logic

### PR DESCRIPTION
**Pull request by jgeorge37**
To test:

1. Create a local PostgreSQL instance (see README). Make sure you add the pgcrypto extension.
2. Change the database URL in the inner .env file to be your local database URL
3. Using probably the psql shell, insert a new row into the account table. This should have the email and the password for a test account. Encrypt the password like so, using the md5 salt thing: `INSERT INTO account (email, password) VALUES ('testEmail@test.com', crypt('testpw', gen_salt('md5')));` 
4. Check that the above worked by using `SELECT * FROM account;`, there should be the email you chose and a jumbled password.
5. Comment out the line that says comment out if using local DB in pg-query.js
6. npm run dev and go to the sign in page. Try to sign in with a bad email/pw combo; there should be a red error message above the submit button
7. sign in with your good user/pw combo, the error message should go away.

### CODE REVIEW

File: acmw-db-app/pages/signin.js [hunk 1]
```diff
@@ -96,24 +103,43 @@ const TextInput = (props) => {
...
         onBlur={() => setActive(false)}
       />
-      <label htmlFor={id} className={error && styles.error}>
-        {error || label}
-      </label>
     </div>
   );
 }
 
 class SignInButton extends React.Component {
```
Comment (SnobbyDragon) [inline acmw-db-app/pages/signin.js:110]: Oh shoot I think I just changed this to using hooks (don't think I merged the ADA-20 yet), but it should be pretty easy to adapt. I think it should be more or less the same...

File: acmw-db-app/pages/signin.js [hunk 2]
```diff
@@ -13,23 +18,31 @@ const SignIn = () => {
...
           <img src="/logo.png" alt="ACM-W Logo" className={styles.logo} />
-          <h1> OSU Database </h1>
+          <h1> ACM-W OSU Database </h1>
           <h2> Please sign in to continue </h2>
           <TextInput
             id="username"
-            label="Username"
-            predicted="Brutus Buckeye"
+            label="Email"
+            predicted="brutus@osu.edu"
```
Comment (SnobbyDragon) [inline acmw-db-app/pages/signin.js:26]: can we have `buckeye.1@osu.edu` i know it doesn't matter that much but :)

File: acmw-db-app/pages/signin.js (hunk 1 above)
Comment (jgeorge37) [reply acmw-db-app/pages/signin.js:110]: Once ADA-20 is merged I can pull those changes and hopefully that will resolve itself

File: acmw-db-app/pages/signin.js (hunk 2 above)
Comment (jgeorge37) [reply acmw-db-app/pages/signin.js]: done

File: acmw-db-app/pages/signin.js (hunk 1 above)
Comment (jgeorge37) [reply acmw-db-app/pages/signin.js:110]: Actually I shouldn't even need to pull unless there are conflicts so I'm asking for approval now

File: acmw-db-app/pages/signin.js [hunk 3]
```diff
@@ -96,24 +103,43 @@ const TextInput = (props) => {
         onFocus={() => setActive(true)}
         onBlur={() => setActive(false)}
       />
-      <label htmlFor={id} className={error && styles.error}>
-        {error || label}
-      </label>
```
Comment (SnobbyDragon) [inline acmw-db-app/pages/signin.js:101]: I think this was the little purple text that said "Username" (will be switched to "Email") and "Password" when the field was focused, so probably put back:
```
<label htmlFor={id}>	
    {label}
</label>
```

Comment (jgeorge37) [reply acmw-db-app/pages/signin.js:101]: oh right yeah my b

**Review by SnobbyDragon — DISMISSED** [review]
Yeet looks great!

**Review by SnobbyDragon — APPROVED** [review]
Nice.

**Pull request merged**